src/test/access-control-transformer.test.js

```diff
@@ -0,0 +1,208 @@
+const assert = require('assert');
+const accessControlTransformer = require('../main/lib/access-control-transformer');
+
+describe('Access control transformer', () => {
+
+  it('should do nothing if AccessControl and UserRoles are not present', async () => {
```

Comment:
Is there a reason that these tests are asynchronous?

Reply:
the only reason is that i'm lazy and did copy paste, of course not needed. Done

Reply:
The best reason